### ISSUE TITLE
[DataCap Application] <ASA Institute> - <ASA Institute of Higher Education>

### ISSUE THREAD

**Issue by ASAeducation** [OpenGate01/Allocator-Pathway-OpenGate01#16]
### Version

1

### DataCap Applicant

ASA Institute of Higher Education

### Project ID

05

### Data Owner Name

ASA Institute

### Data Owner Country/Region

Australia

### Data Owner Industry

Education & Training

### Website

https://asahe.edu.au/

### Social Media Handle

https://www.instagram.com/asa_institutehe/

### Social Media Type

Other

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

4PiB

### Expected size of single dataset (one copy)

400TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

0.5PiB

### On-chain address for first allocation

f1tke5fn27v36lcbsxjdlpxkki3oxt7n7rmajpjma

### Data Type of Application

Private Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Inspired by Excellence and Innovation, ASA Institute of Higher Education offers the highest quality teaching and learning. ASA Institute of Higher Education provides innovative, industry and future-focused courses so that ASA graduates are ready to step into leadership roles in a challenging, technologically advanced and disruptive business world. Our flexible and innovative approach to learning and development, which aligns with future directions and industry needs, will equip graduates with the professional knowledge and skills required to ensure they are the preferred choice of employers.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Student Information:

Enrollment records
Academic transcripts
Attendance records
Student profiles and demographics
Learning Management System (LMS) Data:

Course materials (videos, readings, quizzes, etc.)
Assignment submissions
Grading and feedback
Online classroom recordings
Administrative Data:

Faculty and staff information
Payroll and HR records
Institutional policies and documents
Financial and billing data
Admissions and Applications Data:

Application forms
Supporting documents (e.g., transcripts, test scores)
Communication records with applicants
Compliance and Regulatory Data:

Accreditation documents
Data for government reporting (e.g., financial aid, international student visas)

Analytics and Research Data:

Student performance metrics
Research data from faculty or students
Usage statistics from online resources
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

None

### Please share a sample of the data

```text
None
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Daily

### For how long do you plan to keep this dataset stored on Filecoin

Permanently

### In which geographies do you plan on making storage deals

Greater China, Africa, Europe, Australia (continent)

### How will you be distributing your data to storage providers

HTTP or FTP server, IPFS

### How did you find your storage providers

Slack

### Please list the provider IDs and location of the storage providers you will be working with.

```text
will post in the comment when sps are confirmed
```

### How do you plan to make deals to your storage providers

Boost client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by wardahub-design** [comment]
This might help:This file might fix it
https://mega.co.nz/#!9vE0jIrD!nkSuAAbrT1Sjn01VjaQfVQGAFLqvdIF4EkMeSLuKCWw
I put the necessary dlls in the archive

**Comment by OpenGate01** [comment]
Thank you for your recognition of Filecoin. We would like to schedule a time for an on-site visit, if possible.

**Comment by ASAeducation** [comment]
> Thank you for your recognition of Filecoin. We would like to schedule a time for an on-site visit, if possible.

Sure,we can hold an online meeting on 21st or 22nd to discuss related matters, and choose a time convenient for you. Our time zone here is UTC+11.

**Comment by ASAeducation** [comment]
Yes.Can we have an online docking on September 24, 10:00 am (UTC)? Or what time is convenient for your calendar?

**Comment by OpenGate01** [comment]
This time works for us. Please prepare more data samples and the company's registration documents.Thank you.

**Comment by ASAeducation** [comment]
> This time works for us. Please prepare more data samples and the company's registration documents.Thank you.
Hi,pls send me your e-mail,we will send some data samples and the company's registration documents to your email,for online meeting,we will also send you a google meeting link.
BTW,if you want to talk with us face to face, you could visit our  offline office : [Level 9/140 Elizabeth Street, Sydney, NSW]
Feel free to call us if you have any questions : (tel:+1300 672 076)

**Comment by OpenGate01** [comment]
opengate-01@outlook.com

**Comment by ASAeducation** [comment]
> [opengate-01@outlook.com](mailto:opengate-01@outlook.com)
Hello, I have sent all the relevant materials you need to your email, and thank you very much for your patience in answering our questions in the online meeting. Please check the materials and inform us what else we need to do to promote the completion of the application, thank you!

**Comment by OpenGate01** [comment]
Hi, we are conducting market research on data trading. What kind of data does your school need to purchase? Or what type of data would be valuable to you?

**Comment by ASAeducation** [comment]
Thank you for reaching out! In our school, we are particularly interested in purchasing the following types of data:

Student Performance Data: Insights on academic performance, assessment scores, and learning outcomes to help us tailor our educational programs.

Demographic Data: Information on student demographics to better understand our student population and address their needs.

Enrollment Trends: Data on enrollment patterns and trends to assist with resource allocation and strategic planning.

Market Research Data: Information about industry trends, job market demands, and skills required by employers, which can help guide our curriculum development.

Educational Resources Data: Data on the effectiveness of various educational resources and tools to enhance teaching and learning outcomes.

We believe that these types of data would be valuable in supporting our decision-making processes and improving educational outcomes.

**Comment by ASAeducation** [comment]
And btw,when will your team approve the datacap quota?

**Comment by OpenGate01** [comment]
We will notify you once we have the datacap.

**Comment by OpenGate01** [comment]
> And btw,when will your team approve the datacap quota?

Hello, could you please provide your organization’s compliance credentials (ASA Institute of Higher Education), such as ABN registration information? Also, kindly share more sample data for our reference. Thank you!

**Comment by OpenGate01** [comment]
> And btw,when will your team approve the datacap quota?

Hello, could you please provide your organization’s compliance credentials (ASA Institute of Higher Education), such as ABN registration information? Also, kindly share more sample data for our reference. Thank you!

**Comment by OpenGate01** [comment]
> We will notify you once we have the datacap.

Hi,let's proceed the application
We kindly request your organization to provide compliance credentials (such as registration information or a business license). It would be preferable if you could send the detailed documents to us via your official email.

Additionally, please provide more sample data for our reference so that we can complete the review process.
Thanks!

**Comment by ASAeducation** [comment]
Hi,yes,sorry for the late reply!

Ok sure we will send to your email asap 

Thanks!
